src/briefcase/integrations/cookiecutter.py

```diff
@@ -51,3 +51,17 @@ def float_blue(obj):
         environment.filters["float_red"] = float_red
         environment.filters["float_green"] = float_green
         environment.filters["float_blue"] = float_blue
+
+
+class TOMLEscape(Extension):
+    """Jinja2 extension to escape strings so TOML don't break."""
+
+    def __init__(self, environment):
+        """Initialize the extension with the given environment."""
+        super().__init__(environment)
+
+        def escape_tag(obj):
```

Comment:
The `tag` name that is used on the Python version comes from the terminology of a Python version tag; a name like "escape_toml" would make more sense here.